ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py

```diff
@@ -12,6 +10,48 @@
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 
 
+# nanmedian
+@handle_frontend_test(
+    fn_tree="numpy.nanmedian",
+    dtype_and_a=statistical_dtype_values(function="nanmedian"),
+    dtype=helpers.get_dtypes("float", full=False, none=True),
```

Comment:
this line is not needed.

Reply:
which line? do you mean (dtype=helpers.get_dtypes("float", full=False, none=True),)

Reply:
which line? do you mean (dtype=helpers.get_dtypes("float", full=False, none=True),)

Reply:
yes